pycodestyle checks failing

```2019-07-11T11:54:06.1683085Z ##[section]Starting: pycodestyle
2019-07-11T11:54:06.1685546Z ==============================================================================
2019-07-11T11:54:06.1685795Z Task         : Command line
2019-07-11T11:54:06.1685941Z Description  : Run a command line script using Bash on Linux and macOS and cmd.exe on Windows
2019-07-11T11:54:06.1686098Z Version      : 2.151.1
2019-07-11T11:54:06.1686264Z Author       : Microsoft Corporation
2019-07-11T11:54:06.1686441Z Help         : https://docs.microsoft.com/azure/devops/pipelines/tasks/utility/command-line
2019-07-11T11:54:06.1686588Z ==============================================================================
2019-07-11T11:54:06.2836404Z Generating script.
2019-07-11T11:54:06.2854628Z ========================== Starting Command Output ===========================
2019-07-11T11:54:06.2884303Z [command]/bin/bash --noprofile --norc /home/vsts/work/_temp/08299e2e-2a44-4a58-a1cf-cee3846300c9.sh
2019-07-11T11:54:06.9341148Z src/benchmarkstt/metrics/core.py:69:110: W291 trailing whitespace
2019-07-11T11:54:06.9342492Z src/benchmarkstt/metrics/core.py:71:1: W293 blank line contains whitespace
2019-07-11T11:54:06.9342961Z src/benchmarkstt/metrics/core.py:72:48: W291 trailing whitespace
2019-07-11T11:54:06.9343324Z src/benchmarkstt/metrics/core.py:73:54: W291 trailing whitespace
2019-07-11T11:54:06.9344398Z src/benchmarkstt/metrics/core.py:78:5: E265 block comment should start with '# '
2019-07-11T11:54:06.9344873Z src/benchmarkstt/metrics/core.py:78:15: W291 trailing whitespace
2019-07-11T11:54:06.9502724Z ##[error]Bash exited with code '1'.
2019-07-11T11:54:06.9513903Z ##[section]Finishing: pycodestyle```

--- a/src/benchmarkstt/metrics/core.py
+++ b/src/benchmarkstt/metrics/core.py
@@ -66,13 +66,13 @@ class WER(Base):
 
     See: https://en.wikipedia.org/wiki/Word_error_rate
 
-    :param mode: WER variant. 'strict' is the default. 'hunt' applies 0.5 weight to insertions and deletions.
-    :param differ_class: For future use.
-
     Insertions, deletions and substitutions are
     identified using the Hunt–McIlroy diff algorithm.
     This algorithm is the one used internally by Python.
     See https://docs.python.org/3/library/difflib.html
+    
+    :param mode: WER variant. 'strict' is the default. 'hunt' applies 0.5 weight to insertions and deletions.
+    :param differ_class: For future use.
     """
 
     # WER modes

--- a/src/benchmarkstt/metrics/core.py
+++ b/src/benchmarkstt/metrics/core.py
@@ -66,16 +66,16 @@ class WER(Base):
 
     See: https://en.wikipedia.org/wiki/Word_error_rate
 
-    :param mode: WER variant. 'strict' is the default. 'hunt' applies 0.5 weight to insertions and deletions. 
+    :param mode: WER variant. 'strict' is the default. 'hunt' applies 0.5 weight to insertions and deletions.
     :param differ_class: For future use.
     
-    Insertions, deletions and substitutions are 
-    identified using the Hunt–McIlroy diff algorithm.  
+    Insertions, deletions and substitutions are
+    identified using the Hunt–McIlroy diff algorithm.
     This algorithm is the one used internally by Python.
     See https://docs.python.org/3/library/difflib.html
     """
 
-    #WER modes 
+    #WER modes
     MODE_STRICT = 'strict'
     MODE_HUNT = 'hunt'
 

--- a/src/benchmarkstt/metrics/core.py
+++ b/src/benchmarkstt/metrics/core.py
@@ -68,14 +68,14 @@ class WER(Base):
 
     :param mode: WER variant. 'strict' is the default. 'hunt' applies 0.5 weight to insertions and deletions.
     :param differ_class: For future use.
-    
+
     Insertions, deletions and substitutions are
     identified using the Hunt–McIlroy diff algorithm.
     This algorithm is the one used internally by Python.
     See https://docs.python.org/3/library/difflib.html
     """
 
-    #WER modes
+    # WER modes
     MODE_STRICT = 'strict'
     MODE_HUNT = 'hunt'
 

